### DISCUSSION TITLE
Native Smart chat for customer help

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5780] [New Feature/Ideas Requests]
![image](https://github.com/inspireui/support/assets/57155882/bff4d750-2139-45b7-99c5-1aa97270251d)

**Comment by osman-kocak** [comment]
@minhcasi

**Comment by chungxon** [comment]
Hi @osman-kocak,

We have supported several intelligent chat services such as

Firebase Realtime chat (native): https://support.inspireui.com/help-center/articles/121/126/109/firebase-realtime-chat
Zoho SalesIQ live chat (native): https://support.inspireui.com/help-center/articles/121/126/211/zoho-salesiq-chat
Other chat options via webview

May I know which one you want to use?

Regards,
Son.

**Reply by osman-kocak** [reply]
tawk.to Live Chat

**Reply by chungxon** [reply]
I checked and found that `tawk.to` does not provide any native solution. You can only try using webview by following this guide https://help.tawk.to/article/how-to-add-the-tawkto-widget-to-your-mobile-application and add it to FluxBuilder

![image](https://github.com/user-attachments/assets/6146029f-2f18-4c48-9355-5718e3b9161d)
